ml-agents/mlagents/trainers/tests/torch/test_networks.py

```diff
@@ -124,52 +124,6 @@ def test_valuenetwork():
             assert _out[0] == pytest.approx(1.0, abs=0.1)
 
 
-@pytest.mark.parametrize("use_discrete", [True, False])
```

Comment:
Is there anything replacing this test?

Reply:
Yes, all this functionality was moved into the action model. All the action model tests replace these.